docs/endpoints/post-identity-map.md

```diff
@@ -9,7 +9,7 @@ import Link from '@docusaurus/Link';
 
 # POST /identity/map
 
-Maps multiple email addresses, phone numbers, or their respective hashes to their raw UID2s and salt bucket IDs. You can also use this endpoint to check for updates to opt-out information.
+Maps multiple email addresses, phone numbers, or their respective hashes to their raw UID2s and <Link href="../ref-info/glossary-uid#gl-salt-bucket">salt bucket</Link> IDs. You can also use this endpoint to check for updates to opt-out information.
 
```

Comment:
Should the link lead to salt bucket id description instead?

Reply:
Yes. Done. Thx.